network/network.go

```diff
@@ -27,6 +27,34 @@ func getVnetClient() network.VirtualNetworksClient {
 	return vnetClient
 }
 
+// CreateVirtualNetwork creates a virtual network
+func CreateVirtualNetwork(ctx context.Context, vnetName string) (vnet network.VirtualNetwork, err error) {
+	vnetClient := getVnetClient()
+	future, err := vnetClient.CreateOrUpdate(
+		ctx,
+		helpers.ResourceGroupName(),
+		vnetName,
+		network.VirtualNetwork{
+			Location: to.StringPtr(helpers.Location()),
+			VirtualNetworkPropertiesFormat: &network.VirtualNetworkPropertiesFormat{
+				AddressSpace: &network.AddressSpace{
+					AddressPrefixes: &[]string{"10.0.0.0/8"},
+				},
+			},
+		})
+
+	if err != nil {
+		return vnet, fmt.Errorf("cannot create virtual network: %v", err)
+	}
+
+	err = future.WaitForCompletion(ctx, vnetClient.Client)
+	if err != nil {
+		return vnet, fmt.Errorf("cannot get the vnet create or update future response: %v", err)
+	}
+
+	return future.Result(vnetClient)
+}
+
```

Comment:
cant we just leave the `CreateVirtualNetworkAndSubnets` ? do we need this func also ?

Reply:
We can leave the func that also creates the subnets